Makefile

```diff
@@ -43,16 +59,6 @@ else
 endif
 DOCKER_TAG = ${DOCKER_NAME}:${DOCKER_VERSION}
 
-# Indicates which version of the UI console is to be embedded
-# in the docker image. If "local" the CONSOLE_LOCAL_DIR is
-# where the UI project has been git cloned and has its
-# content built in its build/ subdirectory.
-# WARNING: If you have previously run the 'docker' target but
-# later want to change the CONSOLE_VERSION then you must run
-# the 'clean' target first before re-running the 'docker' target.
-CONSOLE_VERSION ?= latest
-CONSOLE_LOCAL_DIR ?= ../../../../../kiali-ui
-
 # Indicates the log level the app will use when started.
 # <4=INFO
 #  4=DEBUG
```

Comment:
Unrelated to PR but are these values both supposed to be 4?

Reply:
yes, weird Go convention.